src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -40,6 +41,26 @@ public function __construct($statusMetadataExtractor = null)
         $this->statusMetadataExtractor = $statusMetadataExtractor ?: new StatusMetadataExtractor();
     }
 
+    /**
+     * Returns the customer ID information of the provided request.
+     *
+     * @param Message $request the request to get its customer ID
+     * @return string the customer ID information
+     */
+    private static function getCustomerIdInfo(Message $request): string
+    {
+        if (method_exists($request, 'getCustomerId')) {
+            return $request->getCustomerId();
+        } elseif (method_exists($request, 'getResourceName')) {
```

Comment:
Could you add a quick comment explaining that it is covering all get requests?

Reply:
Again, I think it's safer not to mention that explicitly. We're likely to forget to update that if anything changes.
I frame the sentence to sound like we provide an example instead.